lib/pages/editor-page.js

```diff
@@ -68,7 +68,10 @@ export default class EditorPage extends AsyncBaseContainer {
 
 		await this.chooseInsertMediaOption();
 		await this.sendFile( newFile );
-		return await this.driver.sleep( 1000 );
+		return await driverHelper.waitTillEnabled(
+			this.driver,
+			by.css( 'button[data-e2e-button="confirm"]' )
+		);
 	}
```

Comment:
`sendFile()` function ends when the media file is selected but it doesn't mean that it is ready to be used. When I was testing, I noticed that even if the media file is selected, the confirmation button gets enabled a bit later. New driverHelper method `waitTillEnabled` waits for the button to be enabled which provides the necessary wait time. As per `wp-media-upload-spec.js`, we don't need to click on the confirmation button but will be nice to wait until it is enabled prior to be clicked if and when needed.